1 HW_Standard/HW_Graveler.yaml

```diff
@@ -20,3 +20,26 @@ nature: Serious
 ivs: {hp: 31, atk: 31, def: 0, spA: 31, spD: 31, spe: 31}
 evs: {hp: 252, atk: 252, def: 0, spA: 252, spD: 252, spe: 0}
 hidden: false
+---
+species: Graveler
+ingamename: Wally Wall
+displayname: Nurikabe
+setname: Halloween
+tags: [halloween]
+item: Protector
+ability: Filter
+moves:
+    - [Rock Slide]
+    - [Will-O-Wisp]
+    - [Iron Defense]
+    - [Block]
+gender: [m]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Dusk]
+shiny: True
+nature: Serious
+ivs: {hp: 31, atk: 0, def: 31, spA: 0, spD: 31, spe: 0}
+evs: {hp: 252, atk: 0, def: 252, spA: 0, spD: 252, spe: 0}

```

Comment:
As I assume the bulk is part of the theme here, I would downgrade to rock throw for balance (will o stops it from being cancer)